src/client/src/models/Scene.js

```diff
@@ -41,41 +41,66 @@ const Scene = {
     return JSON.parse(localStorage.getItem('scene-list'))
   },
 
+  findSceneIndex: function(sceneid) {
+    // get current list
+    const list = JSON.parse(localStorage.getItem('scene-list'))
+
+    for (var i = 0; i < list.length; i++) {
+      if (sceneid == list[i].id) {
```

Comment:
use `===` whenever you can with JS to avoid type coercion

Reply:
Tired to and it didn't work.